bundles-lib/src/main/java/org/apache/metron/bundles/BundleClassLoaders.java

```diff
@@ -119,8 +120,10 @@ public static void init(final FileSystemManager fileSystemManager, final List<Fi
       if (bundleClassLoaders != null) {
         throw new IllegalStateException("BundleClassloader already exists");
       }
```

Comment:
Sorry I missed this earlier.  The `initContext = null` has to be after this line, otherwise you might destroy only half of the info.